src/plugins/map/MQ2Map.cpp

```diff
@@ -263,15 +263,27 @@ PLUGIN_API void InitializePlugin()
 	{
 		MapFilterOption& option = MapFilterOptions[i];
 
-		// If it's the CampRadius or PullRadius, let's not get the last on/off state, lets assume it's off so we don't draw a circle at 0, 0.
-		if (_stricmp(option.szName, "CampRadius") && _stricmp(option.szName, "PullRadius"))
+		option.Enabled = GetPrivateProfileInt("Map Filters", option.szName, option.Default, INIFileName);
```

Comment:
I know this code was just moved, but it looks like a boolean (GetPrivateProfileBool vs Int)